plugins/varnish/varnish4_multigraph

```diff
@@ -0,0 +1,453 @@
+#!/usr/bin/python3 -tt
+# -*- coding: utf-8 -*-
+
+""" Munin plugin to monitor Varnish 4 status.
+
+Copyright 2016, Kim B. Heino, b@bbbs.net, Foobar Oy
+License GPLv2+
+
+This is based heavily on varnish4_ plugin from Munin contrib
+by Kristian Lyngstol <kristian@bohemians.org> / Redpill Linpro AS.
+
+#%# capabilities=autoconf
+#%# family=auto
+"""
+# pylint: disable=invalid-name
+# pylint: enable=invalid-name
+
+import json
+import subprocess
+import sys
+
+
+def run_binary(arg):
+    """Run binary and return output."""
+    try:
+        cmd = subprocess.Popen(
+            arg, shell=False, close_fds=True, bufsize=-1,
+            stdout=subprocess.PIPE, stderr=subprocess.PIPE)
+        outdata, dummy_errdata = cmd.communicate()
+    except OSError:
+        return ''
+    return outdata.decode('utf-8', 'ignore')
+
+
+def get_values():
+    """Run varnishstat and parse it's output."""
+    output = run_binary(['/usr/bin/varnishstat', '-j'])
+    try:
+        return json.loads(output)
+    except (TypeError, ValueError):
+        return {}
+
+
+def autoconf():
+    """Print "yes" or "no"."""
+    status = 'yes' if get_values() else 'no'
```

Comment:
It would be great, if you could distinguish between *varnishstat is missing* and *execution of varnishstat failed*.